Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,7 +25,8 @@ class BrowserViewController: UIViewController {
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
     private var screenshotHelper: ScreenshotHelper!
-    private var homePanelIsInline: Bool = false
+    private var homePanelIsInline = false
+    private var searchLoader: SearchLoader!
```

Comment:
Why are you making this implicitly unwrapped? Why are we creating it in init and not just when its needed?